src/controllers/cohort.js

```diff
@@ -1,10 +1,14 @@
-import { createCohort, getAllCohorts } from '../domain/cohort.js'
+import {
+  createCohort,
+  getAllCohorts,
+  getCohortById,
+  updateCohortNameByID
+} from '../domain/cohort.js'
 import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 
 export const create = async (req, res) => {
   try {
-    const createdCohort = await createCohort()
-
+    const createdCohort = await createCohort(req.body.name)
```

Comment:
It might be a good idea to check if the name isn't empty before passing it  off to the `createCohort` function